9.2/test/run

```diff
@@ -255,6 +255,66 @@ function run_tests() {
   fi
 }
 
+function run_replication_test() {
+  echo "Testing master-slave replication"
+  local cluster_args="-e POSTGRESQL_ADMIN_PASSWORD=pass -e POSTGRESQL_MASTER_USER=$PGUSER -e POSTGRESQL_MASTER_PASSWORD=$PASS"
+  local max_attempts=30
+
+  # Run the PostgreSQL master
+  docker run $cluster_args -p 5432  \
+    -d --cidfile ${CIDFILE_DIR}/master.cid $IMAGE_NAME run-postgresql-master >/dev/null
+  local master_ip
+  master_ip=$(get_container_ip master.cid)
+
+  # Run the PostgreSQL slave
+  docker run $cluster_args -e POSTGRESQL_MASTER_IP=${master_ip} \
+    -d --cidfile ${CIDFILE_DIR}/slave.cid $IMAGE_NAME run-postgresql-slave >/dev/null
```

Comment:
We should have somehow more "abstract" 'start_master' action and 'start_slave' functions
to be able easily change number of slaves.  But this might be extracted in follow up patches.